### PR TITLE
Drop and component fix

### PR DESCRIPTION
**Pull request by antond15**
- Fixed not opening drop inventory. This was caused by changed value in `PlayerData.id`.
- Fixed extended clip attachment for Combat MG Mk II.
- Removed some old debug stuff.

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -672,7 +672,6 @@ local function updateInventory(items, weight)
 	end
 
 	client.setPlayerData('inventory', PlayerData.inventory)
-	TriggerEvent('ox_inventory:updateInventory', changes)
```
Comment (antond15) [inline client.lua:675]: I am not 100% sure if this is left from testing `add` and `remove` options or if it's something with a similar purpose as `ox_inventory:currentWeapon`, for example.

Comment (thelindat) [reply client.lua:675]: This is for other resources to listen for changes; can be used for add/remove functionality and metadata changes.

File: client.lua [hunk 2]
```diff
@@ -706,7 +705,7 @@ RegisterNetEvent('ox_inventory:createDrop', function(drop, data, owner, slot)
 		drops[drop] = data
 	end
 
-	if owner == PlayerData.id and invOpen and #(GetEntityCoords(PlayerData.ped) - data.coords) <= 1 then
+	if owner == GetPlayerServerId(PlayerData.id) and invOpen and #(GetEntityCoords(PlayerData.ped) - data.coords) <= 1 then
```
Comment (thelindat) [inline client.lua:708]: Did I end up storing PlayerData.source? If not then add that during init and use here.

File: modules/items/server.lua [hunk 3]
```diff
@@ -121,39 +121,6 @@ local itemFormat = [[
...
-		local Players = ESX.GetPlayers()
-		for i = 1, #Players do
-			local i = Players[i]
-			--if not IsPlayerAceAllowed(i, 'command.refresh') then
-				local inv, ped = Inventory(i), GetPlayerPed(i)
-				local hash, curWeapon = GetSelectedPedWeapon(ped)
-				if not ignore[hash] then
-					curWeapon = Utils.GetWeapon(hash)
-					if curWeapon then
-						local count = 0
```
Comment (thelindat) [inline modules/items/server.lua:136]: This is part of detecting weapon cheats that I need to go back and rewrite; old code is left behind for quick reference.

**Comment by thelindat** [comment]
Would be best to drop the commits rather than recommitting what was removed

**Pull request closed without merging**